WordPress/src/main/java/org/wordpress/android/sharedlogin/JetpackSharedLoginFlag.kt

```diff
@@ -8,5 +8,5 @@ class JetpackSharedLoginFlag @Inject constructor(
     private val jetpackSharedLoginFeatureConfig: JetpackSharedLoginFeatureConfig,
     private val buildConfigWrapper: BuildConfigWrapper
 ) {
-    fun isEnabled() = jetpackSharedLoginFeatureConfig.isEnabled() && buildConfigWrapper.isJetpackApp
+    fun isEnabled() = buildConfigWrapper.isJetpackApp
```

Comment:
Same question: was the feature config removed for testing or are we handling it differently?

Reply:
Same as: https://github.com/wordpress-mobile/WordPress-Android/pull/17398#discussion_r1011027792